app/filtering.js

```diff
@@ -362,6 +362,13 @@ function registerPermissionHandler (session, partition) {
       return
     }
 
+    // Always allow fullscreen if setting is ON
+    const alwaysAllowFullscreen = module.exports.alwaysAllowFullscreen() === 'alwaysAllow'
```

Comment:
Instead of using `'alwaysAllow'`, can you import `settingsEnums.js` instead and check against `fullscreenOption. ALWAYS_ALLOW`?  Always good to avoid hardcoding (and use the enum value) when possible